api/task/pipelines.go

```diff
@@ -60,7 +60,7 @@ func SetClient(computeClient *compute.Client) {
 }
 
 func submitPipeline(datasets []string, step *description.FullySpecifiedPipeline) (string, error) {
-	return sr.SubmitPipeline(client, datasets, nil, nil, step)
+	return sr.SubmitPipeline(client, datasets, nil, nil, step, false)
```

Comment:
I'm worried that no pipelines will be cached now. We definitely want a lot of them to still be cached (PCA, clustering, etc.).

Reply:
Right - clustering and MI ranking are the two that benefit the most from caching, and can slow down model runs since the ta2 process requests serially.

Reply:
that is a whoops. Good catch. I will add the param to submitPipeline and fix that.

Reply:
Fixed on the recent push.